packages/admin-ui/src/filters/multiselect-filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { tag } from '@vtex/admin-ui-react'
+import { Checkbox } from '../components/Checkbox'
+import { Filter } from './filter'
+import { Option } from './option'
+import type { FilterItem, UseMultipleFilterReturn } from './filter.state'
+
+export function MultiselectFilter(props: MultiselectFilterProps) {
+  const {
+    state: { listState, appliedValues },
+    state,
+  } = props
+
+  const firstSelectedItemLabel =
+    appliedValues?.length > 1 ? `${appliedValues[0]},` : `${appliedValues[0]}`
+
+  const remainingSelectedItemsCount =
+    appliedValues?.length > 1 && `+${appliedValues.length - 1}`
```

Comment:
```suggestion
  const firstSelectedItemLabel = `${appliedValues[0]}`

  const remainingSelectedItemsCount =  appliedValues.length > 1 ? `, +${appliedValues.length - 1}` : null
```

Reply:
there's no way of putting the comma on the second text and having the effect I want.

The effect: "value, +2" (and the value part is on a separate div)

Reply:
This is the current implementation:
```code
<tag.span csx={{ color: '$primary' }}>
  <tag.span
    csx={{
      textOverflow: 'ellipsis',
      overflow: 'hidden',
      whiteSpace: 'nowrap',
      maxWidth: '300px',
      marginX: '$s',
    }}
    >
      {firstSelectedItemLabel}
  </tag.span>
  {remainingSelectedItemsCount}
</tag.span>
```
I think if you change the `marginX` property to be on its parent, this should work properly.  Something like this:

`<tag.span csx={{ color: '$primary' , marginX: '$s' }}>`

This is just a suggestion, feel free to accept or not